src/ews_xsd.erl

```diff
@@ -443,15 +468,15 @@ print_schema_stats(Schema) ->
     AttributeGroups = wh:get_children(Schema, "attributeGroup"),
     Notations = wh:get_children(Schema, "notation"),
     Annotations = wh:get_children(Schema, "annotation"),
-    io:format("e: ~p, s: ~p, c: ~p, g: ~p, a: ~p, ag: ~p, n: ~p, an: ~p~n",
-              [length(Elements),
-               length(SimpleTypes),
-               length(ComplexTypes),
-               length(Groups),
-               length(Attributes),
-               length(AttributeGroups),
-               length(Notations),
-               length(Annotations)]).
+    logger:debug("e: ~p, s: ~p, c: ~p, g: ~p, a: ~p, ag: ~p, n: ~p, an: ~p~n",
+                 [length(Elements),
+                  length(SimpleTypes),
+                  length(ComplexTypes),
+                  length(Groups),
+                  length(Attributes),
+                  length(AttributeGroups),
+                  length(Notations),
+                  length(Annotations)]).
```

Comment:
Using `?LOG_DEBUG` would save some evaluations here.

Reply:
This code runs like once and never again. So do we really need to optimize it?

Reply:
No we don't, I didn't realise that context.